client/src/Root.tsx

```diff
@@ -8,14 +8,34 @@ import { EditorLogin } from 'editor-pages/EditorLogin';
 import { Result } from 'client-pages/Result';
 import { ResultReport } from 'client-pages/ResultReport';
 import { StatementVerifier } from 'client-pages/StatementVerifier';
+import { VerdictType } from 'components/StatementEvaluation';
+import { IResult } from 'models/Result';
+import { IReport } from 'models/Report';
+
+const testResultData: IResult = {
+  statement: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit',
+  verdict: VerdictType.TRUTH,
+  probability: 97,
+  id: 'chyba-niepotrzebne-bo-jest-w-url',
+};
+
+const testReportData: IReport = {
+  id: 'lorem',
+  category: 'Finanse',
+  comment: 'Zaraz mi odpadnie łapa',
+  date: '29.12.2020',
+  politician: 'Mateusz Morawiecki',
+  reporter: 'Boris Bierezowski',
```

Comment:
To być może kwestia niefortunnego nazewnictwa w interface IReport, ale reporter generalnie traktowany jest jako e-mail.